README.md

```diff
@@ -42,13 +42,14 @@ Note that all third-party extensions are removed upon upgrading.
 
 ## Adding extensions after installation
 
+You need before to install [Composer](https://getcomposer.org/download/).
```

Comment:
Composer is installed and available in `/var/www/$app/composer.phar`, hence the `cd /var/www/$app` beforehand. No need to install it again.